libethashseal/genesis/mainNetworkTest.cpp

```diff
@@ -27,7 +27,7 @@ R"E(
 		"daoHardforkBlock": "0x1d4c00",
 		"EIP150ForkBlock": "0x259518",
 		"EIP158ForkBlock": "0x28d138",
-		"metropolisForkBlock": "0xfffffffffffffff",
+		"metropolisForkBlock": "0x2dc6c0",
```

Comment:
3 000 000  metropolis transition block. just for the time being (in test network)